Source/WebDriver/HTTPServer.cpp

```diff
@@ -28,9 +28,14 @@
 
 namespace WebDriver {
 
-HTTPServer::HTTPServer(HTTPRequestHandler& requestHandler)
-    : m_requestHandler(requestHandler)
+HTTPServer::HTTPServer(WeakRef<HTTPRequestHandler>&& requestHandler)
+    : m_requestHandler(WTFMove(requestHandler))
```

Comment:
I'm not sure this is correct, I think usually you want to pass a reference to the object and create the weak reference internally, so 

```cpp
HTTPServer::HTTPServer(HTTPRequestHandler& requestHandler)
    : m_requestHandler(requestHandler)
```

would work just fine if `m_requestHandler` is a `WeakRef<HTTPRequestHandler>`.